Content.Server/Explosion/EntitySystems/TriggerSystem.TimedCollide.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Linq;
+using Content.Server.Explosion.Components;
+using Content.Server.Explosion.EntitySystems;
+using Robust.Shared.Physics.Dynamics;
+
+namespace Content.Server.Explosion.EntitySystems;
+
+public sealed partial class TriggerSystem
+{
+    //Stores the entity as active so the event can resolve for each UID colliding with this entity.
+    private readonly List<EntityUid> Active = new();
```

Comment:
Just make a new component called "ActiveTriggerOnTimedCollideComponent" and use that instead.

Reply:
I'm unsure how to go about this. From what I understand the list is supposed to be there to store each entity individually so the event triggers for those specific entities uniquely.

Reply:
I'm still a bit lost. If I make the new component, I would need to have two components on a yaml?

I'm just not processing how I'd write this and include this and such.

Reply:
Should be fixed now I think.